doc/zh-cn/install-dependencies.md

```diff
@@ -40,15 +40,21 @@ CentOS 7
 wget http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
 sudo rpm -ivh epel-release-latest-7.noarch.rpm
 
+# 安装 etcd
```

Comment:
Please update this doc in English, too.

Reply:
It is copied from the English document,no need to update the English version.